sky/shell/mac/sky_window.mm

```diff
@@ -70,12 +70,29 @@ - (NSString*)skyInitialLoadURL {
               .infoDictionary objectForKey:@"org.domokit.sky.load_url"];
 }
 
+- (NSString*)skyInitialBundleURL {
+  return [[NSBundle mainBundle] pathForResource:@"app" ofType:@"skyx"];
+}
+
 - (void)setupAndLoadDart {
   auto interface_request = mojo::GetProxy(&_sky_engine);
   self.platformView->ConnectToEngine(interface_request.Pass());
 
-  mojo::String string(self.skyInitialLoadURL.UTF8String);
-  _sky_engine->RunFromNetwork(string);
+  NSString *endpoint = self.skyInitialBundleURL;
```

Comment:
This code does *not* belong in sky_window.  NSApplicationDelegate  didFinishLaunching?

https://developer.apple.com/library/mac/documentation/Cocoa/Reference/NSApplicationDelegate_Protocol/#//apple_ref/occ/intfm/NSApplicationDelegate/applicationDidFinishLaunching: